Private/Utilities.ps1

```diff
@@ -34,17 +34,31 @@ url = https://github.com/3shapeAS/dockerbuild-pwsh.git
     $configData | Out-File -FilePath "$dotGitPath/config" -Encoding ascii
 }
 
-function Add-Postfix {
+function Add-RegistryPostfix {
     param (
-        [Parameter(Mandatory=$true)]
-        [String] $Data,
+        [String] $Registry,
         [ValidateNotNullOrEmpty()]
         [String] $Postfix = '/'
     )
 
-    $trimmedData = $Data.Trim()
-    if ( -Not $trimmedData.EndsWith($Postfix) ) {
-            $trimmedData += $Postfix
+    # Do nothing if $Data is empty / writespace
+    if ([String]::IsNullOrWhiteSpace($Registry)) {
+        return ''
+    }
+
+    $trimmedRegistry = $Registry.Trim()
+    if ( -Not $trimmedRegistry.EndsWith($Postfix) ) {
+            $trimmedRegistry += $Postfix
     }
-    $trimmedData
+    $trimmedRegistry
+}
+
+function Test-DockerDigest {
+    param (
+        [Parameter(Mandatory=$true)]
+        [String] $Digest
+    )
+
+    $shaPrefix = 'sha256:'
+    return $Digest.StartsWith($shaPrefix) -and $Digest.Length -eq ($shaPrefix.Length + 64)
```

Comment:
64 seems to be a bit of magic number, perhaps you could extract it to a constant and give it a more explanatory name?

Reply:
👍